### DISCUSSION TITLE
Using Jotai with class components

### DISCUSSION THREAD

**Discussion by sreeraj-wd** [pmndrs/jotai#299] [Q&A]
How to use jotai with **class components** ? Is there any code snippet available somewhere showing the same.

**Comment by dai-shi** [comment]
Can you wrap your class component with a function component?
```js
const FuncComponent = () => {
  const [foo, setFoo] = useAtom(fooAtom)
  return (
    <ClassComponent foo={foo} setFoo={setFoo} />
  )
}
```

**Reply by jatinhemnani01** [reply]
Using Jotain in class component is possible?

**Reply by dai-shi** [reply]
Do you mean wrapping is not a solution? Then, not possible.
